docs/guides/admin/docs/workflowoperationhandlers/speechtotext-woh.md

```diff
@@ -10,13 +10,16 @@ The speech to text operation can be used to generate subtitles for Videos or Aud
 engines available, [Whisper](../modules/transcription.modules/whisper.md) and 
 [Vosk](../modules/transcription.modules/vosk.md). The subtitles file format ist WebVTT.
 
+This operation is designed to only work with one source file per flavor. If you want to use multiple source files in one
+flavor, you need to set the same tag to each element.
```

Comment:
This seems overcomplicated. In particular, forcing users to add tags to tracks if they want to translate multiple at the same time. Why not just "This operation will attempt to translate all tracks in the given `source-flavor`. To restrict which tracks to translate, add tags to the respective tracks and specify them in `source-tags`"?

Reply:
Yes, you are right. That line is confusing. This PR aims to work like any other Workflow.
If source flavors and tags are meet, it should select the correct files.